### DISCUSSION TITLE
program exits after cli.run() command

### DISCUSSION THREAD

**Discussion by acutis-expeditus** [cloud-custodian/cloud-custodian#9226] [General]
Am calling c7n_org.cli.run(...) from Python, but it's observed that code exits after this call.
I have print statement right after this call, but that isn't executed.

I do see that there is no error in executing the run method and the success value is also true.

**Comment by ajkerrigan** [comment]
You can certainly run c7n/c7n-org from Python, but it's not guaranteed to be stable and isn't supported. There's more discussion in #8553

**Reply by acutis-expeditus** [reply]
Ok.

Am running into another issue
"Error running policy, Resource:aws.ec2 not loaded"
